.maintain/frame-weight-template.hbs

```diff
@@ -66,3 +66,31 @@ impl<T: frame_system::Trait> WeightInfo for SubstrateWeight<T> {
 	}
 	{{/each}}
 }
+
+// For backwards compatibility and tests
+impl WeightInfo for () {
```

Comment:
I would really like to get rid of this `()` impl. Now that the `SubstrateWeight` lives inside the pallet's module why can't we use this for testing? You can specify `SubstrateWeight<T=RocksDb>` if you want to make `T` optional for easier specification in tests.

Reply:
I think we will deprecate this in the future, but for now, with the wrong communication, can cause a lot of pain for devs

Reply:
I see. Looking forward to this.